docs/API_reference.md

```diff
@@ -571,8 +571,7 @@ The 'dataframe' format transforms that data structure into a pandas DataFrame, w
 Get list of raw data files and (optionally) download raw data.
 
 [*Optional parameter*]:
-**dataName**:  restricts the search results to raw data files e.g. orders,orderItems.
-Multiple raw data files can be separated by comma.
+**dataName**:  restricts the search results to raw data files e.g. orders, orderItems.
```

Comment:
The title of the commit doesn't really explain what you *fixed* here? To me it just looks like you removed a helpful comment.

Reply:
Searching multiple files with a comma-separated search string doesn't work.

Instead of 
`{'dataName':'orderItems,orderItemAmounts','createdAtTimestamp':'1644590630' }`

This seems to work:
`{'dataName':['orderItems','orderItemAmounts'],'createdAtTimestamp':'1644590630' }`
(I'll fix that)